src/hera.cpp

```diff
@@ -263,17 +265,14 @@ evm_result evm_execute(
   evm_result ret;
   memset(&ret, 0, sizeof(evm_result));
 
-  try {
-    heraAssert(msg->gas >= 0, "Negative startgas?");
-
-    ExecutionResult result;
-    result.gasLeft = (uint64_t)msg->gas;
+  hera_instance* hera = static_cast<hera_instance*>(instance);
```

Comment:
These changes seem not be needed?

Reply:
this was moved from line 276 so that the WasmVM object can be instantiated with the correct options, without needing a redundant declaration of hera_instance.